test/spec/provider/cloud-element-templates/fixtures/connectors.json

```diff
@@ -277,5 +277,92 @@
         }
       }
     ]
+  },
+  {
```

Comment:
(minor) I realized that we use `connectors.json` as filename. However, this really is general C8 element templates, I guess connectors could be a bit confusing as filename isn't it?

Reply:
Well it started with `connectors`, but it evolved 😉 I will rename the file.

Reply:
Renamed via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/564/commits/5cb36c227028a2489c440a5bf82f3771f0312822